src/PSRule/Pipeline/PipelineContext.cs

```diff
@@ -183,6 +180,12 @@ internal void Import(IResource resource)
             }
         }
 
+        private void TrackIssue(IResource resource)
+        {
+            //if (resource.TryValidateResourceAnnotation())
+            //    _TrackedIssues.Add(new ResourceIssue(resource.Kind, resource.Id, ResourceIssueType.MissingApiVersion));
+        }
+
```

Comment:
Is this just a stub method where we track issues issues for later?

Reply:
@ArmaanMcleod Thanks Armaan. Yep. I expect we'll use it again soon.